front/app/api/analytics/types.ts

```diff
@@ -24,7 +24,7 @@ export interface QuerySchema {
         };
   };
   groups?: string | string[];
-  aggregations?: AggregationsConfig;
+  aggregations?: AggregationsConfig | AggregationsConfig[];
```

Comment:
@jamesspeake You mentioned needing to fix these typings, but I don't see any console errors or anything when I just use `AggregationsConfig | AggregationsConfig[]` :thinking: Have I missed something though?